config/versioning.py

```diff
@@ -2,12 +2,19 @@
 
 
 def get_git_sha():
+    value_from_env = os.getenv("PLATFORM_TREE_ID", "")
+    if value_from_env:
+        return value_from_env
+
     base_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
     head_path = os.path.join(base_dir, ".git", "HEAD")
 
-    with open(head_path, "r") as fp:
-        head = fp.read().strip()
+    try:
+        with open(head_path, "r") as fp:
+            head = fp.read().strip()
+    except FileNotFoundError:
+        return ""
```

Comment:
Could we mark this as temporary or maybe  get_git_sha?

Reply:
There's no way to do this lazily, unfortunately - as the config is needed to initialise the sentry client. I think leaving it as a blank string if it cannot be identified is a safe enough fallback.

Perhaps when we move to AWS, there's be something similar we can get from the environment too.